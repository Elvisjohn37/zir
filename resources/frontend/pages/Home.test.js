describe('<Home />', () => {
	it.todo('Should match snapshot');

	it.todo('Should display loading indicator for main banner');

	it.todo('Should display the main banner');

	it.todo('Should display loading indicator for new games list inside the banner section');

	it.todo('Should display new games list inside the banner section');

	it.todo('Should have user control if the new game list items are greater than 2');

	it.todo('Should display button to play and new games wording');

	it.todo('');
});
