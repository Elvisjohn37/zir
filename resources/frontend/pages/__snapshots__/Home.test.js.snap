// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> Should match snapshot 1`] = `
<DocumentFragment>
  <div
    classes="{\\"root\\":\\"container\\"}"
    data-testid="Container"
    maxwidth="lg"
  >
    <div
      classes="{\\"root\\":\\"mainContentContainer\\"}"
      data-testid="Container"
      maxwidth="lg"
    >
      <div
        classes="{\\"root\\":\\"container1\\"}"
        data-testid="Container"
      >
        <div
          class="announcementCard"
          data-testid="Card"
        >
          <div
            data-testid="ErrorBoundary"
          >
            <div
              ads="[{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/liveBlackJackAnnouncement.png\\",\\"url\\":null,\\"type\\":2}]"
              data-testid="Placeholder"
              height="112"
              width="460"
            />
            <div
              banner="{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/liveBlackJackAnnouncement.png\\",\\"url\\":null,\\"type\\":2}"
              class="announcement"
              data-testid="Announcement"
              langstate="{\\"active\\":\\"en\\",\\"available\\":[\\"en\\",\\"zh\\",\\"ko\\",\\"th\\"],\\"disabled\\":[\\"th\\"]}"
            />
          </div>
        </div>
        <div
          class="gameTypePreviewCard"
          data-testid="Card"
        >
          <div
            data-testid="ErrorBoundary"
          >
            <div
              ads="[{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/liveBlackjackPromo.webm\\",\\"url\\":null,\\"type\\":3},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/liveBaccaratPromo.webm\\",\\"url\\":null,\\"type\\":3},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/liveRoulettePromo.webm\\",\\"url\\":null,\\"type\\":3},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/liveSicboPromo.webm\\",\\"url\\":null,\\"type\\":3},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/slotsPromo.webm\\",\\"url\\":null,\\"type\\":3},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/cardGamesPromo.webm\\",\\"url\\":null,\\"type\\":3},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/tableGamesPromo.webm\\",\\"url\\":null,\\"type\\":3},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/videoGamesPromo.webm\\",\\"url\\":null,\\"type\\":3}]"
              data-testid="Placeholder"
              height="369"
              width="460"
            />
            <div
              banners="[{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/liveBlackjackPromo.webm\\",\\"url\\":null,\\"type\\":3},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/liveBaccaratPromo.webm\\",\\"url\\":null,\\"type\\":3},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/liveRoulettePromo.webm\\",\\"url\\":null,\\"type\\":3},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/liveSicboPromo.webm\\",\\"url\\":null,\\"type\\":3},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/slotsPromo.webm\\",\\"url\\":null,\\"type\\":3},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/cardGamesPromo.webm\\",\\"url\\":null,\\"type\\":3},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/tableGamesPromo.webm\\",\\"url\\":null,\\"type\\":3},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/videoGamesPromo.webm\\",\\"url\\":null,\\"type\\":3}]"
              class="gameTypePreview"
              data-testid="GameTypePreview"
              langstate="{\\"active\\":\\"en\\",\\"available\\":[\\"en\\",\\"zh\\",\\"ko\\",\\"th\\"],\\"disabled\\":[\\"th\\"]}"
            />
          </div>
        </div>
      </div>
      <div
        classes="{\\"root\\":\\"container2\\"}"
        data-testid="Container"
      >
        <div
          class="gamePromotionalPreview1Card"
          data-testid="Card"
          variant="outlined"
        >
          <div
            data-testid="ErrorBoundary"
          >
            <div
              ads="[{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/casino2.png\\",\\"url\\":\\"https://www.sbobet.com/games?cat=TableGames\\",\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/casino3.png\\",\\"url\\":\\"https://www.sbobet.com/games?cat=TableGames\\",\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/casino4.png\\",\\"url\\":\\"https://www.sbobet.com/games?cat=TableGames\\",\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/casino5.png\\",\\"url\\":\\"https://www.sbobet.com/games?cat=TableGames\\",\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/casino6.png\\",\\"url\\":\\"https://www.sbobet.com/games?cat=TableGames\\",\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/casino7.png\\",\\"url\\":\\"https://www.sbobet.com/games?cat=TableGames\\",\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/mobile1.png\\",\\"url\\":null,\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/mobile2.png\\",\\"url\\":null,\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/mobile3.png\\",\\"url\\":null,\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/sportsbook1.png\\",\\"url\\":\\"https://www.sbobet.com/\\",\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/sportsbook2.png\\",\\"url\\":\\"https://www.sbobet.com/\\",\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/sportsbook3.png\\",\\"url\\":\\"https://www.sbobet.com/\\",\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/table1.png\\",\\"url\\":null,\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/table2.png\\",\\"url\\":null,\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/texaspoker1.png\\",\\"url\\":\\"https://www.kuncimerah.com/\\",\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/texaspoker2.png\\",\\"url\\":\\"https://www.kuncimerah.com/\\",\\"type\\":4}]"
              data-testid="Placeholder"
              height="146"
              width="460"
            />
            <div
              banners="[{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/casino2.png\\",\\"url\\":\\"https://www.sbobet.com/games?cat=TableGames\\",\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/casino3.png\\",\\"url\\":\\"https://www.sbobet.com/games?cat=TableGames\\",\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/casino4.png\\",\\"url\\":\\"https://www.sbobet.com/games?cat=TableGames\\",\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/casino5.png\\",\\"url\\":\\"https://www.sbobet.com/games?cat=TableGames\\",\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/casino6.png\\",\\"url\\":\\"https://www.sbobet.com/games?cat=TableGames\\",\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/casino7.png\\",\\"url\\":\\"https://www.sbobet.com/games?cat=TableGames\\",\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/mobile1.png\\",\\"url\\":null,\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/mobile2.png\\",\\"url\\":null,\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/mobile3.png\\",\\"url\\":null,\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/sportsbook1.png\\",\\"url\\":\\"https://www.sbobet.com/\\",\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/sportsbook2.png\\",\\"url\\":\\"https://www.sbobet.com/\\",\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/sportsbook3.png\\",\\"url\\":\\"https://www.sbobet.com/\\",\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/table1.png\\",\\"url\\":null,\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/table2.png\\",\\"url\\":null,\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/texaspoker1.png\\",\\"url\\":\\"https://www.kuncimerah.com/\\",\\"type\\":4},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/texaspoker2.png\\",\\"url\\":\\"https://www.kuncimerah.com/\\",\\"type\\":4}]"
              class="gamePromotionalPreview1"
              data-testid="GamePromotionalPreview1"
              langstate="{\\"active\\":\\"en\\",\\"available\\":[\\"en\\",\\"zh\\",\\"ko\\",\\"th\\"],\\"disabled\\":[\\"th\\"]}"
            />
          </div>
        </div>
        <div
          class="contactPreviewCard"
          data-testid="Card"
          variant="outlined"
        >
          <div
            data-testid="ErrorBoundary"
          >
            <div
              ads="[{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/customerSupport.png\\",\\"url\\":null,\\"type\\":5}]"
              data-testid="Placeholder"
              height="93"
              width="460"
            />
            <div
              banner="{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/customerSupport.png\\",\\"url\\":null,\\"type\\":5}"
              class="contactPreview"
              data-testid="ContactPreview"
              langstate="{\\"active\\":\\"en\\",\\"available\\":[\\"en\\",\\"zh\\",\\"ko\\",\\"th\\"],\\"disabled\\":[\\"th\\"]}"
            />
          </div>
        </div>
        <div
          class="gamePromotionalPreview2Card"
          data-testid="Card"
          variant="outlined"
        >
          <div
            data-testid="ErrorBoundary"
          >
            <div
              ads="[{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/3populargamesminor1.png\\",\\"url\\":null,\\"type\\":6},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/3populargamesminor2.png\\",\\"url\\":null,\\"type\\":6},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/3populargamesminor3.png\\",\\"url\\":null,\\"type\\":6},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/3populargamesminor4.png\\",\\"url\\":null,\\"type\\":6},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/3populargamesminor5.png\\",\\"url\\":null,\\"type\\":6},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/3populargamesminor6.png\\",\\"url\\":null,\\"type\\":6},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/3populargamesminor7.png\\",\\"url\\":null,\\"type\\":6},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/3populargamesminor8.png\\",\\"url\\":null,\\"type\\":6},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/3populargamesminor5.png\\",\\"url\\":null,\\"type\\":6},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/3populargamesminor6.png\\",\\"url\\":null,\\"type\\":6},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/3populargamesminor7.png\\",\\"url\\":null,\\"type\\":6},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/3populargamesminor8.png\\",\\"url\\":null,\\"type\\":6}]"
              data-testid="Placeholder"
              height="146"
              width="460"
            />
            <div
              banners="[{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/3populargamesminor1.png\\",\\"url\\":null,\\"type\\":6},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/3populargamesminor2.png\\",\\"url\\":null,\\"type\\":6},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/3populargamesminor3.png\\",\\"url\\":null,\\"type\\":6},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/3populargamesminor4.png\\",\\"url\\":null,\\"type\\":6},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/3populargamesminor5.png\\",\\"url\\":null,\\"type\\":6},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/3populargamesminor6.png\\",\\"url\\":null,\\"type\\":6},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/3populargamesminor7.png\\",\\"url\\":null,\\"type\\":6},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/3populargamesminor8.png\\",\\"url\\":null,\\"type\\":6},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/3populargamesminor5.png\\",\\"url\\":null,\\"type\\":6},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/3populargamesminor6.png\\",\\"url\\":null,\\"type\\":6},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/3populargamesminor7.png\\",\\"url\\":null,\\"type\\":6},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/3populargamesminor8.png\\",\\"url\\":null,\\"type\\":6}]"
              class="gamePromotionalPreview2"
              data-testid="GamePromotionalPreview2"
              langstate="{\\"active\\":\\"en\\",\\"available\\":[\\"en\\",\\"zh\\",\\"ko\\",\\"th\\"],\\"disabled\\":[\\"th\\"]}"
            />
          </div>
        </div>
        <div
          class="bankingOptionCard"
          data-testid="Card"
          variant="outlined"
        >
          <div
            data-testid="ErrorBoundary"
          >
            <div
              ads="[{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/mastercard.jpg\\",\\"url\\":null,\\"type\\":7},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/maestro.jpg\\",\\"url\\":null,\\"type\\":7},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/visaelectron.jpg\\",\\"url\\":null,\\"type\\":7},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/visa.jpg\\",\\"url\\":null,\\"type\\":7},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/mb.jpg\\",\\"url\\":null,\\"type\\":7},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/neteller.jpg\\",\\"url\\":null,\\"type\\":7},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/bankwire.jpg\\",\\"url\\":null,\\"type\\":7}]"
              data-testid="Placeholder"
              height="50"
              width="460"
            />
            <div
              banners="[{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/mastercard.jpg\\",\\"url\\":null,\\"type\\":7},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/maestro.jpg\\",\\"url\\":null,\\"type\\":7},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/visaelectron.jpg\\",\\"url\\":null,\\"type\\":7},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/visa.jpg\\",\\"url\\":null,\\"type\\":7},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/mb.jpg\\",\\"url\\":null,\\"type\\":7},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/neteller.jpg\\",\\"url\\":null,\\"type\\":7},{\\"bannerImg\\":\\"http://rso-stg.stry628c.com/banner/bankwire.jpg\\",\\"url\\":null,\\"type\\":7}]"
              class="bankingOption"
              data-testid="BankingOptions"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    classes="{\\"root\\":\\"footerMenuContainer\\"}"
    data-testid="Container"
    maxwidth="lg"
  >
    <div
      class="menuBar"
    >
      <div
        class="menuLinks"
      >
        <div
          data-testid="ErrorBoundary"
        >
          <div
            class="footerMenu"
            data-testid="FooterMenu"
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
